src/update_agent/actor.rs

```diff
@@ -100,7 +114,7 @@ impl Handler<RefreshTick> for UpdateAgent {
         });
 
         // Process state machine refresh ticks sequentially.
-        ctx.wait(update_machine);
```

Comment:
I am unsure whether this `wait` should be converted into a `spawn`, or directly dropped by returning the `update_machine` chain. But I have a feeling this may be ok this way.

Reply:
I've experimented with directly returning the `update_machine` chain in my toy example, and it behaves as expected, so I think dropping it is fine (no need for `ctx.spawn`).